From bf5a7f284dab813f30bae688eddfa5cc4b7aea35 Mon Sep 17 00:00:00 2001
From: "Liu,Dalei" <daleix.liu@intel.com>
Date: Fri, 13 Nov 2015 10:45:23 +0800
Subject: [PATCH 02/28] imx227: Add a VGA 30fps option

Add a VGA 30fps option in the imx227 video setting res.

Change-Id: I0597cd4e4e25ff612d90bf40201b4b01204c5300
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-9279
Original-Tracked-On: #H1604036344
Signed-off-by: Liu,Dalei <daleix.liu@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/84798
Reviewed-by: HaiqiangX Yao <haiqiangx.yao@intel.com>
Approver: Bin B Han <bin.b.han@intel.com>
Reviewed-by: Bin B Han <bin.b.han@intel.com>
Build: Dessert M BXT RVP (SC)
Build: Dessert M COHO (SC)
Integrator: WeiX Meng <weix.meng@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Signed-off-by: Yang Fan F <fan.f.yang@intel.com>
Reviewed-on: https://android.intel.com:443/440488
---
 drivers/external_drivers/camera/drivers/media/i2c/imx/imx227.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/imx/imx227.h b/drivers/external_drivers/camera/drivers/media/i2c/imx/imx227.h
index 76d6198..ccb6a7a 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/imx/imx227.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/imx/imx227.h
@@ -689,6 +689,11 @@ struct imx_resolution imx227_res_video[] = {
 				.lines_per_frame = 0x0302,
 			},
 			{
+				.fps = 30,
+				.pixels_per_line = 0x0E70,
+				.lines_per_frame = 0x0C08,
+			},
+			{
 			}
 		},
 	},
-- 
1.9.1

